src/vm/jitinterface.cpp

```diff
@@ -10914,27 +10939,7 @@ void CEEJitInfo::reserveUnwindInfo(BOOL isFunclet, BOOL isColdCode, ULONG unwind
 
```

Comment:
@parjong we should not add the compact encoding, since it is not used by managed code on any of the platforms. @jkotas has explained that to me.